### PR TITLE
Milestone 8 : Portfolio Details popup window

### PR DESCRIPTION
**Pull request by michael-duke**
##  Portfolio milestone 8 : details popup window

In this milestones I have : 
- Used basic JavaScript syntax.
- Used JavaScript to manipulate DOM elements.
- Used JavaScript events.
- Understood how to parse a Figma design to create a UI.
- Use objects, arrays to store and access data.
- Collaborated with coding partner, and worked in team.
- Followed JavaScript best practices, and ESLint's rules of keeping depedency of files.

### CODE REVIEW

File: styles/projects.css [hunk 1]
```diff
@@ -102,7 +102,151 @@
...
+  width: 100%;
+  height: 80vh;
+  padding: 1rem;
+  background-color: #fff;
+  border: 0.0625rem solid #dfe1e6;
+  border-radius: 1rem;
+  overflow: scroll;
+  animation: zoom-in 0.45s ease-out;
+}
+
```
Comment (1GORDON) [inline styles/projects.css:138]: Great job team 🤝

- [ ] You may try to make your design responsive enough on your mobile popup screen. Your design is not responsive and it makes parts of the information hidden
- Your design
Mobile popup

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @michael-duke & @peterboro ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Descriptive Pull Request ✔️
- Professional Readme ✔️
- Linters are passing ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles/projects.css (hunk 1 above)
Comment (michael-duke) [reply styles/projects.css:138]: @1GORDON  can kindly let me know which screen-size you used for mobile

File: js/popup-detail.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  modal.appendChild(projectContainerGrid);
+  modalBackdrop.appendChild(modal);
+  document.body.appendChild(modalBackdrop);
+}
+
+const onPopup = (e) => {
+  const {
+    path: [, , project],
+  } = e;
+  const id = +project.id.slice(-1);
```
Comment (wuletawwonte) [inline js/popup-detail.js:166]: - [ ] Please make sure to use the correct array or object destructuring syntax. The `See Project` popup is not opening due to an error in these lines. Please fix the error and make sure the popup works before requesting a review.

**Review by wuletawwonte — CHANGES_REQUESTED** [review]
# Hi @michael-duke & @peterboro 👋🏿👋🏿👋🏿  

Good job so far!
There are some issues that you still need to work on to go to the next project! You have only a few more steps and you are done! 💪 💪 💪 

giphy

## TO HIGHLIGHT: 
- Your project is professional ✔️ 
- Correct Github Flow ✔️
- Your PR has a proper title and description ✔️
- Nice commit history ✔️
- Well structured files ✔️ 

## Required Changes ♻️
_Please check the comments under the review as well._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: js/popup-detail.js (hunk 2 above)
Comment (michael-duke) [reply js/popup-detail.js]: Can you kindly let me know what browser you are using @wuletawwonte Thank you 😄

**Comment by michael-duke** [comment]
Thank you @mwanawabangona  and Happy coding to all, this has been great @wuletawwonte  and @1GORDON  aswell.

Cheers 😄

**Pull request merged**